packages/tools/generator-component/generators/app/templates/src/components/Skeleton.vue

```diff
@@ -1,36 +1,36 @@
 <template>
     <div
-        :data-theme="theme"
         :class="$style['c-<%= name.class %>']"
-        data-test-id='<%= name.class %>-component'>
+        data-test-id="<%= name.class %>">
         {{ copy.text }}
     </div>
 </template>
 
 <script>
-import { globalisationServices } from '@justeat/f-services';
-import tenantConfigs from '../tenants';<% if(needsTestingApiMocks) { %>
-import <%= name.filename%>ServiceApi from '../services/<%= name.filename%>ServiceApi';<%}%>
-
+<% if (config.needsComponentTranslations) { %>import { globalisationServices } from '@justeat/f-services';
+import tenantConfigs from '../tenants';
+<%
+}
+if (config.needsTestingApiMocks) { %>import <%= name.filename%>ServiceApi from '../services/<%= name.filename%>ServiceApi';
+<%
+} %>
 export default {
     name: '<%= name.component %>',
     components: {},
     props: {
-        locale: {
+<% if (config.needsComponentTranslations) { %>        locale: {
             type: String,
             default: ''
         }
-    },
+<%} %>    },
     data () {
```

Comment:
could be a case where data has no return value if i18n is not selected

Reply:
Done